### DISCUSSION TITLE
be significant

### DISCUSSION THREAD

**Discussion by JiangnanPsalter** [community/community#60225] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Feedback

Everyone is here to search for **the meaning of creation (innovation and ideas)** and **the significance of maintaining a high level (in order not to down).** and useful resources (or information).
Of course, our life can also be significant, and we really are.

**Comment by GhostOf0days** [comment]
I checked your repos and saw you wanted to go to MIT (you said in a comment on one of your repos). I highly suggest looking into [MIT's summer programs](https://mitadmissions.org/apply/prepare/summer/) for high schoolers if you still are. I highly suggest looking at MIT's Research Science Institute (RSI). It will give you an edge on other first year applicants. Transferring to MIT is very, very, very difficult. It is much harder than first year admission. I know many people who got rejected for transfer there, including kids from schools like CMU and CalTech. Please make your first year application count as much as possible, and have unique ECs (extracurriculars).

**Reply by GhostOf0days** [reply]
My best advice here is high impact ones that branch out of your community and help people in a fairly significant way.

**Reply by JiangnanPsalter** [reply]
First of all, thank you very much for your very useful suggestions. The link you shared has had a profound impact on me. Secondly, I want to achieve my goal in a natural way. I don't want to go to the sky in one step because I don't want my strength to be out of place. This may be a very long journey. However, everything has a predetermined exact number.

**Reply by JiangnanPsalter** [reply]
Dear Ghost, as you can see, I will receive a Bachelor's degree in Artificial Intelligence in our country next year. It is expected to help me complete my software Master of Engineering degree in China, and I really intend to do so. I hope to pursue a PhD in Mathematics at MIT because my pursuit is in this field, and that's exactly what I think. There's no need to hide anything from you, it's correct. Give me a hug

**Reply by GhostOf0days** [reply]
Ah I see. Please make sure you have published research for getting into grad school at MIT. I'm still a sophomore undergrad, so I'm sorry I can't give better advice here.

**Reply by JiangnanPsalter** [reply]
Anyway, I am very grateful to you. You must be a great hacker

**Reply by GhostOf0days** [reply]
Haha no. I'm just a CS student. My interests are AI/deep learning, web dev, quantitative finance, and entrepreneurship.

**Reply by JiangnanPsalter** [reply]
Maybe.

**Reply by GhostOf0days** [reply, marked as answer]
Anyways, best of luck for grad school applications!

**Reply by majed08** [reply]
good look👍

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
